javascript/ecmascript-2015/symbol/symbol-special-properties.md

```diff
@@ -31,42 +31,42 @@ Symbols behave differently to other primitives such as strings and numbers.
 Lets imagine for this section we declare the following object:
 
 ```javascript
-var obj = {}
-obj["company"] = "Enki"
-obj[Symbol()]="You wont see me";
+var obj = {};
+obj["company"] = "Enki";
+obj[Symbol()] = "You wont see me";
 ```
 
-You will find symbols are ignored in **for..in** loops:
+You will find symbols are ignored in `for..in` loops:
 
 ```javascript
-for(var i in obj){
-console.log(i)
+for (var i in obj) {
+  console.log(i);
 }
 //company
 ```
 
-Symbols do not show up in **object.keys**
+Symbols do not show up in `object.keys`:
 
 ```javascript
 Object.keys(obj);
 //["company"]
 ```
 
-Symbols are not returned from a call to **getOwnPropertyNames**:
+Symbols are not returned from a call to `getOwnPropertyNames`:
 
 ```javascript
 Object.getOwnPropertyNames(obj);
 //["company"]
 ```
 
-**JSON.stringify** ignores symbols:
+`JSON.stringify` ignores symbols:
 
 ```javascript
 JSON.stringify(obj);
 //"{"company":"enki"}"
 ```
 
-Note you can retrieve symbols with **getOwnPropertySymbols**
```

Comment:
No need for "Note", just say "You can retrieve..."

Reply:
done